iruby.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest', '~> 5.6'
 
   s.add_runtime_dependency 'bond', '~> 0.5'
-  s.add_runtime_dependency 'rbczmq', '~> 1.7'
```

Comment:
Since we removed it from the dependencies - we have to print a message if the loading of either one fails.